composer.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "treehouselabs/behat-common",
+  "description": "Common Behat contexts to use in our Symfony projects",
+  "license": "proprietary",
+  "type": "library",
+  "minimum-stability": "dev",
+  "prefer-stable": true,
+  "require": {
+    "php": ">=5.4",
+    "behat/behat": "~3.0",
+    "behat/mink": "~1.6",
+    "behat/mink-extension": "~2.0",
+    "behat/mink-browserkit-driver": "~1.2",
+    "behat/symfony2-extension": "~2.0",
+    "doctrine/orm": "~2.2",
+    "doctrine/common": "~2.2",
+    "doctrine/data-fixtures": "~1.1",
+    "nelmio/alice": "dev-master",
+    "phpunit/phpunit": "~4.7",
+    "swiftmailer/swiftmailer": ">=4.2.0,~5.0",
+    "symfony/dependency-injection": "~2.2",
+    "symfony/http-kernel": "~2.2",
+    "symfony/property-access": "~2.2",
```

Comment:
Deze mag je ook hoger inzetten: 2.7 is een LTS, ik stel voor om daar op te gaan zitten.